EcalTiming/plugins/EcalTimingCalibProducer.cc

```diff
@@ -505,9 +519,10 @@ EcalTimingCalibProducer::Status EcalTimingCalibProducer::duringLoop(const edm::E
 #endif
 	// If we got less than the minimum recHits, continue
 	if(eventTimeMap_.size() < _recHitMin) return kContinue;
-	if(timeEB.num() < 4) return kContinue;
-	if(timeEEM.num() < 4 && timeEEP.num() < 4) return kContinue;
+	if(timeEB.num() + timeEEM.num() + timeEEP.num() < _minEntries) return kContinue;
```

Comment:
The reason was to require an event with enough activity. So the original cut was separately in EB and EE such that you need a certain number of channels active to have a good chance that was a collision event and not just random noise. It is not needed now, it was used during the first debugging period.

-- 
 							Shervin